samples/presentation-player/player-sample-1/output/player.js

```diff
@@ -63,44 +112,69 @@ SampleSkin = function(player)
 			}
 		}, this);
 	}
-
-	window.leftPosition = 0;
-	window.scale = 1;
-	if (!presentation.settings().appearance().fitToWindow())
-	{
-		document.body.style.overflow = "auto";
-		window.onresize = function() {
-			var top = Math.max(0, Math.floor((document.documentElement.clientHeight - playerHeight) / 2));
-			playerView.style.top = top + "px";
-
-			var left = Math.max(0, Math.floor((document.documentElement.clientWidth - playerWidth) / 2));
-			playerView.style.left = left + "px";
-
-		};
-	}
-	else
-	{
-		document.body.style.overflow = "hidden";
-		playerView.style.webkitTransformOrigin = "0 0";
-		playerView.style.msTransformOrigin = "0 0";
-		playerView.style.transformOrigin = "0 0";
-
-		window.onresize = function() {
-			window.scale = Math.min(document.documentElement.clientWidth / playerWidth, document.documentElement.clientHeight / playerHeight);
-			playerView.style.webkitTransform = "scale(" + window.scale + ")";
-			playerView.style.msTransform = "scale(" + window.scale + ")";
-			playerView.style.transform = "scale(" + window.scale + ")";
-
-			var scaledHeight = Math.round(playerHeight * scale);
-			var top = Math.max(0, Math.floor((document.documentElement.clientHeight - scaledHeight) / 2));
-			playerView.style.top = top + "px";
-
-			var scaledWidth = Math.round(playerWidth * scale);
-			window.leftPosition = Math.max(0, Math.floor((document.documentElement.clientWidth - scaledWidth) / 2));
-			playerView.style.left = window.leftPosition + "px";
-		};
-	}
-	window.onresize();
-
-	document.onselectstart = function() { return false; }
-};
\ No newline at end of file
+	return narrationViewElement;
+}
+
+/**
+ * @param {number} playerWidth
+ * @param {number} playerHeight
+ * @param {boolean} sidePanelAtLeft
+ * @return {HTMLElement}
+ */
+function initializePlayerDiv(playerWidth, playerHeight, sidePanelAtLeft)
+{
+	var /** HTMLElement */ playerDiv = document.getElementById("player");
+	setStyleSize(playerDiv, playerWidth, playerHeight);
+	playerDiv.className = sidePanelAtLeft ? "sidePanelAtLeft" : "sidePanelAtRight";
+	return playerDiv;
+}
+
+/**
+ * @param {string} title
+ */
+function initializeTitle(title)
+{
+	var titlePanel = document.getElementById("titlePanel");
+	titlePanel.innerHTML = title;

```

Comment:
It is better to encode html entities, because title may contain special html characters which may break html markup

Reply:
changed to innerText here and in other places